eZ/Bundle/PlatformInstallerBundle/src/Command/InstallPlatformCommand.php

```diff
@@ -61,6 +62,12 @@ protected function configure()
             InputArgument::REQUIRED,
             'The type of install. Available options: ' . implode(', ', array_keys($this->installers))
         );
+        $this->addOption(
+            'disable-index',
```

Comment:
skip-indexation ?

Reply:
Yeah, naming is always difficult :)

skip-indexing sounds good enough for me, wdyt?

Reply:
@ilukac`skip-indexing` is fine